pythonforandroid/recipes/greenlet/__init__.py

```diff
@@ -6,4 +7,15 @@ class GreenletRecipe(PythonRecipe):
     url = 'https://pypi.python.org/packages/source/g/greenlet/greenlet-{version}.tar.gz'
     depends = [('python2', 'python3crystax')]
 
+    def get_recipe_env(self, arch=None, with_flags_in_cc=True):
+        env = super(GreenletRecipe, self).get_recipe_env(arch, with_flags_in_cc)
+        # sets linker to use the correct gcc (cross compiler)
+        env['LDSHARED'] = env['CC'] + ' -pthread -shared -Wl,-O1 -Wl,-Bsymbolic-functions'
```

Comment:
Otherwise the default host linker was picked up rather than the ARM cross compiler.

Reply:
Also addressed in https://github.com/kivy/python-for-android/pull/1361